classes/person.rb

```diff
@@ -0,0 +1,37 @@
+require_relative 'nameable'
+require_relative 'capitalize_decorator'
+require_relative 'trimmer_decorator'
```

Comment:
- [ ] The name of this decorator has a typo, and it's causing an error when requiring it here, As required by the previous reviewer, please rename the module to be `capitalize_decorator.rb` instead of `capittilize_decorator.rb`.

![image](https://github.com/evansnyamekye/school-library/assets/68796232/84b72dfe-599b-4d14-915a-0d05a6f8ce33)